### ISSUE TITLE
Is there a way to add a timeout to spike simulation

### ISSUE THREAD

**Issue by taoliug** [riscv-software-src/riscv-isa-sim#345]
I am running spike in the baremetal mode. Sometimes, spike run infinitely because of a bad program. In this case, is there a way to terminate the spike simulation after a certain number of instructions?

**Comment by aswaterman** [comment]
No, but you can use the `timeout` program to effect something similar (e.g. `timeout 5 spike myprogram`)
